extensions/functions/onnx/onnx.conf

```diff
@@ -0,0 +1,2 @@
+# include onnx c api
+/usr/local/onnx/lib
```

Comment:
Users need to install lib here?

Reply:
This is not required, but it can also be installed in this directory. Perhaps I should remove it? @ngjaying